wordpress-mobile/WordPress-Android
# Issue/10648 material web preview

Fixes #10648

This PR moves Web Preview screens to the material theme and adds dark mode support.

![comparison_webpreview](https://user-images.githubusercontent.com/728822/73146032-647cda80-4065-11ea-98b4-600593d4b084.png)

To test:
- Check Site, Post and Theme previews and make sure they look ok in both light and dark themes.

The theme can be changed from app settings.

PR submission checklist:
